File: dockers/python3/script_loader.py
from ast import literal_eval as make_tuple
import json
from script import *;

def main():
    with open('./parameters.json', 'r') as f:
        arg_json = json.loads(f.read())
        arg = make_tuple(arg_json[0]['value'])
        if arg:
            run(arg)
        else:
            print('Parameter error.')

if __name__ == '__main__':
    main()

File: dockers/python3/script_loader.js
let run = require('./script')
let fs = require('fs')
fs.readFile('parameters2.json','utf8',function (err, data) {
    if(err) console.log(err);
    let parameter=JSON.parse(data);
    let allparameter = {}
    parameter.forEach(
        pr=>{
            allparameter[pr.name]=pr.type==='NUMBER'?Number(pr.value):pr.value
        }
    )
    run(allparameter)
});
